### ISSUE TITLE
Suggestion for AOP enhancement

### ISSUE THREAD

**Issue by willrennie** [salesagility/SuiteCRM#100]
It is easy to manually turn a contact into a Portal User, however, is it possible to automate this process so that all Contacts, as they get created, become Portal Users? It would be normal for customers (who become contacts) to automatically receive support access details. It is going to be tiresome to have to manually do this to all Contacts one by one.

**Comment by EZS-JD** [comment]
What I would love also is for the whole thing to work the other way around as well: Have users register via Joomla which would have them appear in Contacts (with cases, etc.) automatically (and potentially add the contact to specific projects, etc.)...

**Comment by ghermans** [comment]
@willrennie  @JimMackin is there any update on this?
The suggestion from @EZS-JD  is a very good enhancement for the AOP :+1:

**Comment by JimMackin** [comment]
@ghermans Unfortunately I don't think there's been any work on this yet. It'll likely not be in 7.5 since this version will focus on AOR reports. If I get time over the weekend I may have a quick look to see how much work it would be to add in the suggestion by @EZS-JD - it shouldn't be too much work however.

**Comment by ghermans** [comment]
@JimMackin just to inform as i am looking forward to this enhancement, did you had any chance to check this ? / do you have any idea when or in what version this will be integrated?

**Comment by JimMackin** [comment]
@ghermans No sorry! Never had a chance to check it out. Can't make any promises as to when I will either.

**Comment by ghermans** [comment]
Thanks for the feedback.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Suggestion for AOP enhancement](https://trello.com/c/11oxrPSs/165-suggestion-for-aop-enhancement)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
